### PR TITLE
Launch the new default browser screen and fix copy

### PR DESCRIPTION
**Pull request by chung-lai**
Change copy and launch the new default browser flow without experiment:

**Updated screenshot with the correct casing on 2nd screen:**

Screen Shot 2022-07-21 at 9 31 50 PM

| Screen 1 | Screen 2 |
| --- | --- |
| Screen Shot 2022-07-21 at 9 01 53 PM | Screen Shot 2022-07-21 at 9 01 56 PM |
| Screen Shot 2022-07-21 at 9 03 43 PM | Screen Shot 2022-07-21 at 9 03 45 PM |
| Screen Shot 2022-07-21 at 9 05 23 PM | Screen Shot 2022-07-21 at 9 05 26 PM |

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### CODE REVIEW

**Review by jedireza — APPROVED** [review]
Code changes look good. I built and test drove locally. Thanks!

**Comment by chung-lai** [comment]
> @chung-lai , is it too late to make the bullets to be sentence casing (vs Title Casing), on screen 2?

Not too late, good catch @hwiatneeva! Will change, thanks

File: Client/Frontend/DefaultBrowserOnboarding/InterstitialVIewModel.swift [hunk 1]
```diff
@@ -153,25 +153,17 @@ class InterstitialViewModel: ObservableObject {
...
+            "Browser without ads or trackers",
+            "Cookie pop-up blocker",
+        ]
     }
 
     func onboardingPageBullets() -> [String] {
         return [
-            "Browse the Web Ad-Free",
-            "Block Trackers, and Pop-ups",
+            "Browse the web ad-free",
```
Comment (jedireza) [inline Client/Frontend/DefaultBrowserOnboarding/InterstitialVIewModel.swift:165]:
```suggestion
            "Browse the Web ad-free",
```

For consideration.

Comment (chung-lai) [reply Client/Frontend/DefaultBrowserOnboarding/InterstitialVIewModel.swift:165]: Interesting, I don't have strong sense about this whether we should capitalize. Will defer to @hwiatneeva :)

**Comment by bikramneeva** [comment]
looks good . good to go on my end

**Pull request merged**